Game2048Base/src/main/java/models/grid2048/Grid2048.java

```diff
@@ -179,7 +186,7 @@ private static Square buildUpdatedSquareByKeyEvent(Square[][] grid, String keyEv
         }
     }
 
-    private static void createRandomTileOnKeyEventHandlerGrid2048(KeyEventHandler keyEventHandler) {
+    public static void createRandomTileOnKeyEventHandlerGrid2048(KeyEventHandler keyEventHandler) {
```

Comment:
Small thing but it's usually nice to put all public methods (entries into the class) above all private methods in the file.